docs/api/puppeteer.page.waitfornetworkidle.md

```diff
@@ -17,12 +17,14 @@ class Page {
 
 ## Parameters
 
-| Parameter | Type                                     | Description                                   |
-| --------- | ---------------------------------------- | --------------------------------------------- |
-| options   | { idleTime?: number; timeout?: number; } | <i>(Optional)</i> Optional waiting parameters |
+| Parameter        | Type                                     | Description                                                        |
```

Comment:
thanks for the contribution but the documentation is now generated from the source code: TypeScript types and the comments on the corresponding methods/types. It needs to be updated there first and this file is automatically generated.